types/figma/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json" ,
+    "rules": {
```

Comment:
Please remove all these rules here.. And fix the errors. New packages cannot override the default rules and need to fix all the errors.
Looking at the figma package it only contains package.json... So i am not sure this is correct package name and types for the npm package.

Reply:
Thanks for the feedback! Please see the description in the PR, this isn't a typings for a normal NPM package.

I removed the extra lint rules as per review in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/37382, but accidentally let the PR close so I had to open a new one.

Reply:
If thats the case you need to change the name of the package so that it doesnt conflict with existing package and fix the header accordingly. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package

Reply:
Even if we create a unique name for this typings, I'm not aware of any mechanism on the NPM side to prevent someone to create a package that conflicts with a DefinitelyTyped typings?

`@types/figma` the most natural name for developers writing Figma plugins. In that sense, the empty `figma` package on NPM is perfect for security purposes. In case someone gets confused and thinks they need to `npm install figma` to make plugins work (they don't), nothing bad will happen.

Reply:
You can keep the `npm-naming` rule to false but please remove `dt-header` disabling from here. Also in the header please add github ids instead of emails to make sure our bot can ping you when things change in the package. Thanks.